images.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"github.com/k0kubun/go-ansi"
+	"github.com/schollz/progressbar/v3"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"slices"
+)
+
+type ImageInfo struct {
+	Path string
+	Info fs.FileInfo
+}
+
+/**
+ * Determine if the given file looks like an image
+ */
+func isImage(fileInfo fs.FileInfo) bool {
```

Comment:
If you wanted to get slightly fancier, you could use [this package](https://pkg.go.dev/mime) to query the MIME type of the files and filter those instead.